botogram/api.py

```diff
@@ -127,7 +127,8 @@ def call(self, method, params=None, files=None, expect=None):
                     reason = "blocked"
 
                 # This happens when the user deleted its account
-                elif status == 403 and "deleted" in message:
+                elif (status == 403 and
+                      "deleted" in message or "deactivated" in message):
```

Comment:
Can you check only for the current error message? The old one isn't relevant anymore.